releng/corundum-src/corundum.properties

```diff
@@ -0,0 +1,303 @@
+# The name of the product for display purposes
+
+product.name = Product

```

Comment:
Are these line endings going to be an issue?

Reply:
Unlikely, have been in there forever